presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/AccumuloQueryRunner.java

```diff
@@ -171,58 +171,29 @@ public static Connector getAccumuloConnector()
         }
 
         try {
-            MiniAccumuloCluster accumulo = createMiniAccumuloCluster();
-            Instance instance = new ZooKeeperInstance(accumulo.getInstanceName(), accumulo.getZooKeepers());
-            connector = instance.getConnector(MAC_USER, new PasswordToken(MAC_PASSWORD));
-            LOG.info("Connection to MAC instance %s at %s established, user %s password %s", accumulo.getInstanceName(), accumulo.getZooKeepers(), MAC_USER, MAC_PASSWORD);
+            FixedHostPortGenericContainer<?> accumuloContainer = new FixedHostPortGenericContainer<>("prestodev/accumulo:latest");
```

Comment:
set a version

(@kokosing do we publish `latest` at all?)

Reply:
https://github.com/prestosql/docker-images/pull/74 will need released and then I will push up a version and that should get the CI to pass. AFAIK `latest` versions are not published.

Reply:
It looks like we do: https://hub.docker.com/layers/prestodev/hdp2.6-hive-kerberized/latest/images/sha256-5831557495ff24f1619325c0a847552a6871376f1b65473664d86ef54a67bc8e?context=explore

@adamjshook you might want to to publish this image to some custom repository to just to see if everything works and then I will do the release so we can update the image here. WDYT?

Reply:
@kokosing Sure, I'm good with that 👍